v19.1/migration-overview.md

```diff
@@ -40,7 +40,7 @@ Above a certain size, many data types such as [`STRING`](string.html)s, [`DECIMA
 
 ## Unsupported data types
 
-CockroachDB does not provide `ENUM` or `SET` data types.
+CockroachDB does not provide [`ENUM`](https://en.wikipedia.org/wiki/Enumerated_type) or `SET` data types.
```

Comment:
This link is to a wikipedia entry that is much broader than the SQL ENUM type, so it's not particularly helpful.